flake.nix

```diff
@@ -31,16 +31,12 @@
         let
           pkgs = import nixpkgs {
             inherit system;
-            config.allowUnfreePredicate = pkg: builtins.elem (lib.getName pkg) [ "pngout" ];
```

Comment:
Since it’s not setting a config anymore, can you change this to `pkgs = nixpkgs.legacyPackages.${system};`? That should save an unnecessary reevaluation of nixpkgs.